### ISSUE TITLE
PUBLIC_URL not working

### ISSUE THREAD

**Issue by msebald** [Awesome-Technologies/synapse-admin#296]
Hello,

I just updated my synapse-admin installation and unfortunately it does not work anymore. Here it runs on the same host as Synapse itself, in the subdirectory /synapse-admin. I tried the default docker-compose.yml, then I used `args` with `- PUBLIC_URL="/synapse-admin"` and I also tried to use the `docker run` command:

`docker run --name synapse-admin -d --restart unless-stopped -p 9090:80 -e "PUBLIC_URL=/synapse-admin" -e "REACT_APP_SERVER=https://matrix.example.com" awesometechnologies/synapse-admin`

PUBLIC_URL has no effect at all. I always get the website that Javascript must be enabled, but if I check the sourcecode, the JS files (and also the manifest file) are loaded from the root of the website:

```
<script src="./static/js/2.92f953cd.chunk.js"></script>
<script src="./static/js/main.da35a7d2.chunk.js"></script>
```

Is there a problem or user problem? ;-)

Regards,
Martin

**Comment by starcraft66** [comment]
`PUBLIC_URL` is a build argument for the Dockerfile, not an environment variable that the app reads at runtime. The image must be rebuilt for the `PUBLIC_URL` to be changed.

**Comment by msebald** [comment]
@starcraft66 I was not able to use the docker-compose.yml file. So I tried a different approach.

I was able to use docker-compose.yml now, I looked a bit deeper into the errors.

The correct syntax is:

```
      args:
        - PUBLIC_URL=/synapse-admin
        - REACT_APP_SERVER=https://matrix.example.com
```

Note the missing quotation marks. At least for `PUBLIC_URL` this was the clue. So I removed it also for `REACT_APP_SERVER`.

So the README file needs to be adjusted.
